assignments/array-methods.js

```diff
@@ -54,30 +54,46 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
 let fullName = [];
+runners.foreach(` ${ runners[i]["first_name"] } ${ runners[i]["last_name"] }`);
```

Comment:
You're mixing up forEach `arr.forEach(element => element` and for loops `for (let i=0; i < arr.length; i++)`. They do the same thing but you can only use one or the other.

Reply:
This line should have looked something like ```runners.forEach( runner => {
    fullName.push(`${runner.first_name} ${runner.last_name}`);
})``` the forEach just goes through the runners array for us. Then inside you do your action. So now full-name would have our first and last names.